packages/code-du-travail-frontend/src/layout/Footer.js

```diff
@@ -32,16 +32,21 @@ const Footer = () => (
           <CategoryTitle>Code du travail numérique</CategoryTitle>
           <List>
             <StyledListItem key="0">
+              <Link route="droit-du-travail">
+                <a>Le droit du travail</a>
+              </Link>
+            </StyledListItem>
+            <StyledListItem key="1">
               <Link route="about">
                 <a>À propos</a>
               </Link>
             </StyledListItem>
-            <StyledListItem key="1">
+            <StyledListItem key="2">
```

Comment:
you can remove the `key` prop it's not needed outside an array

Reply:
nice 👍